pkg/statdb/statdb_test.go

```diff
@@ -50,7 +50,12 @@ func testDatabase(ctx context.Context, t *testing.T, sdb statdb.DB) {
 			UptimeSuccessCount: currUptimeSuccess,
 		}
 
-		stats, err := sdb.Create(ctx, nodeID, nodeStats)
+		meta := &statdb.Meta{
+			Wallet: "",
+			Email:  "",
+		}
```

Comment:
`meta := &statdb.Meta{}` is an equivalent shorter notation.

Reply:
I'll replace these